tests/servers/tests.py

```diff
@@ -114,16 +115,16 @@ def test_view(self):
         Ensure that the LiveServerTestCase serves views.
         Refs #2879.
         """
-        f = self.urlopen('/example_view/')
-        self.assertEqual(f.read(), b'example view')
+        with contextlib.closing(self.urlopen('/example_view/')) as f:
```

Comment:
is there is reason not to have `self.urlopen()` return `f.read()` instead of just `f`? Just seems like the tests could be less verbose that way.